CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -402,9 +402,10 @@ def checkConfigurationSheet(wb, ws, configurationSheetColumnName, wsOut, verbose
     return missingSheetList
 
 
-def validate_workbook(wb, messages, args=None):
-    if not wb:
-        return
+def validate_workbook(file_obj, messages, args=None):
```

Comment:
this could also accept both arguments. either a workbook or file object and if its a file object it could load the workbook.

Reply:
Thought of that and decided to keep it one.